### ISSUE TITLE
ALTHOLD mode never sent in LTM when MAG mode is enabled

### ISSUE THREAD

**Issue by krzysztofmatula** [iNavFlight/inav#1597]
I think the MAG mode is usually a long-term setting, while ALTHOLD rather short-term.
I propose changing the order of if-s in ltm.c, so ALTHOLD is reported first, then MAG mode only if ALTHOLD is not enabled.

Is:
```
    else if (FLIGHT_MODE(HEADFREE_MODE) || FLIGHT_MODE(MAG_MODE))
        lt_flightmode = 11;
    else if (FLIGHT_MODE(NAV_ALTHOLD_MODE))
        lt_flightmode = 8;
```
should be:
```
    else if (FLIGHT_MODE(NAV_ALTHOLD_MODE))
        lt_flightmode = 8;
    else if (FLIGHT_MODE(HEADFREE_MODE) || FLIGHT_MODE(MAG_MODE))
        lt_flightmode = 11;
```

I'll prepare PR if you agree.

**Comment by digitalentity** [comment]
Sure, please go ahead.

**Comment by krzysztofmatula** [comment]
Fixed in 1.7.
